### PR TITLE
Replace unpinned actions with pinned action

### PR DESCRIPTION
**Pull request by stacklokbot**
This is a Minder automated pull request.

This pull request replaces references to actions by tag to references to actions by SHA.

Verifies that any actions use pinned tags
Pinning an action to a full length commit SHA is currently the only way to use
an action as an immutable release. Pinning to a particular SHA helps mitigate
the risk of a bad actor adding a backdoor to the action's repository, as they
would need to generate a SHA-1 collision for a valid Git object payload.
When selecting a SHA, you should verify it is from the action's repository
and not a repository fork.

For more information, see
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

### CODE REVIEW

File: .github/workflows/releaser.yml [hunk 1]
```diff
@@ -108,7 +105,7 @@ jobs:
       actions: read # To read the workflow path.
       id-token: write # To sign the provenance.
       contents: write # To add assets to a release.
-    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.9.0
```
Comment (rdimitrov) [inline .github/workflows/releaser.yml:111]: @JAORMX @jhrozek - I recall we should exclude some of these (one had a problem with the digest reference)

Comment (JAORMX) [reply .github/workflows/releaser.yml:111]: @rdimitrov you are right, https://github.com/stacklok/frizbee/blob/main/.frizbee.yml#L5

Comment (JAORMX) [reply .github/workflows/releaser.yml:111]: This is configurable via minder though, it's just about setting up the right `excludes` in there.

Comment (rdimitrov) [reply .github/workflows/releaser.yml:111]: until it is configured we should hold off merging this one then 👍

Comment (jhrozek) [reply .github/workflows/releaser.yml:111]: This is our configuration:
```
  - type: actions_check_pinned_tags
    def:
      exclude:
        - slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@1.9.0
```
It also marks the frizbee repo as succeeded, so currently I'm unsure what's going on

Comment (rdimitrov) [reply .github/workflows/releaser.yml:111]: Also strange that the CLA got triggered, I remember we added the bot when we did the first enrolment. cc: @dussab

Comment (JAORMX) [reply .github/workflows/releaser.yml:111]: @jhrozek aha! the tag doesn't match. The configuration uses `1.9.0` while we have configured `v1.9.0` in the action.

Comment (JAORMX) [reply .github/workflows/releaser.yml:111]: @jhrozek the right tag seems to be `v1.9.0`

Comment (rdimitrov) [reply .github/workflows/releaser.yml:111]: hawk-eye-ozz in action! 🦅 😃

Comment (jhrozek) [reply .github/workflows/releaser.yml]: This was fixed by modifying the profile. Thanks for the sharp eyes!

**Comment by jhrozek** [comment]
Going to close this PR and let minder create a new one to get a nice screen for a blog post :-)

**Pull request closed without merging**